crates/ruff_python_semantic/src/model.rs

```diff
@@ -2229,6 +2235,14 @@ bitflags! {
         /// [PEP 257]: https://peps.python.org/pep-0257/#what-is-a-docstring
         const ATTRIBUTE_DOCSTRING = 1 << 26;
 
+        /// The model is in a `typing.Annotated` annotation.
+        ///
+        /// For example, the model could be visiting `int` or "A" in
+        /// ```python
+        /// def f(x: Annotated[int, "A"]): ...
+        /// ```
+        const TYPING_ANNOTATED_PEP593 = 1<<27;
```

Comment:
oof... but I have been trying to think of a better name for 10 minutes and I've got nothing 😆

Reply:
Seems like a reasonable name given other variants although I leave this up to you to decide, maybe you come up with a better name in the morning ;)

Reply:
Aside, it would be useful to provide a link to PEP 593 in the documentation above @dylwil3